cypress.config.ts

```diff
@@ -0,0 +1,35 @@
+import path from 'path';
+import { defineConfig } from 'cypress';
+
+export default defineConfig({
+  e2e: {
+    baseUrl: 'http://localhost:3001',
+    retries: 1,
+    supportFile: 'cypress/support/e2e.ts',
```

Comment:
e2e are failing because not tests are found.
It seems that's because the default pattern for finding test files is this: `cypress/e2e/**/*.cy.{js,jsx,ts,tsx}` ([reference](https://docs.cypress.io/guides/references/configuration#Testing-Type-Specific-Options)).

We could add something like this:
```suggestion
    supportFile: 'cypress/support/e2e.ts',
    specPattern: 'cypress/integration/**/*.{js,jsx,ts,tsx}',
```

Reply:
Thanks!

Actually let me push a commit to restructure the files as well, following the structure and naming conventions.

Reply:
Restructured things here: https://github.com/commercetools/merchant-center-application-kit/pull/2713/commits/a8b9f9672f617540898a295778cc266d25344a07

This is also nicer as we get proper icons in VSCode:
<img width="227" alt="image" src="https://user-images.githubusercontent.com/1110551/179200509-f69ada35-c525-4e2d-9529-ca135419c9bb.png">